src/components/LevelOfStudySelect/LevelOfStudySelect.tsx

```diff
@@ -0,0 +1,39 @@
+import { Autocomplete } from '@mui/material'
+import { useState } from 'react'
+
+import LEVELOFSTUDIES from '@/constants/levelofstudies'
+import useFuse from '@/hooks/useFuse'
+import { TextField } from '@/styles/custom'
+
+import type { Dispatch, SetStateAction } from 'react'
+
+type Props = {
+  value: string | null
+  setValue: Dispatch<SetStateAction<string | null>>
+}
+
+const LevelOfStudySelect = ({ value, setValue }: Props) => {
+  const [query, setQuery] = useState('')
+  const results = useFuse(LEVELOFSTUDIES, query)
+
+  return (
```

Comment:
would recommend using a dropdown select field instead of an autocomplete since there are ~10 options. See `age` dropdown component.

prob can set undergrad (3+ yrs) as default option